Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -102,14 +118,35 @@ void BrowsingContextGroup::addFrameProcessAndInjectPageContextIf(FrameProcess& p
         }
 #endif
         set.add(WTFMove(newRemotePage));
+    };
+
+    if (process.isSlushBucket()) {
+        ASSERT(m_slushBucketSites.size() == 1);
```

Comment:
I don't see how this assertion would hold true for general browsing.

Reply:
I think it should because when we're creating `FrameProcess` for the slush bucket, we've only added one site to the set but I'll change it to the loop over m_slushBucketSites anyway since it's equally easy to do.